src/server/scripts/World/npcs_special.cpp

```diff
@@ -1090,6 +1094,14 @@ class npc_injured_patient : public CreatureScript
                     me->SetHealth(me->CountPctFromMaxHealth(25));
                     break;
             }
+
+            // emulate vanilla WoW's 200ms spell batching rate for maximum authenticity!
+            // https://www.wowhead.com/classic/news/spell-batching-in-classic-wow-290910
```

Comment:
Okay, but the question is. Did this exist in 3.3.5?

Reply:
So, I don't currently have a character in WotLK Classic to test with that has First Aid 225. All the videos I can find have the person _successfully_ completing the quest, which means that not enough guys die to tell how the timing works. Worse, the live version doesn't give you decreasing healthbars - the soldiers all have "full" health and then up and die after a set timer.

Given that I didn't have a way to find out for sure, I just replicated the existing functionality that relied on the vanilla tick rate of every 200ms, presuming that whoever wrote that 7+ years ago was about right for timing. I figure working and close is better than utterly broken.

If anyone has the inclination, a video of the quest getting started and then just waiting while all the soldiers die (with them in view) would be really helpful in replicating the actual quest functionality.

Reply:
> Okay, but the question is. Did this exist in 3.3.5?

Dang it, you made me want to figure this out. I power-leveled first aid for a character on WotLK Classic and figured out the exact timing.

- Critically Injured: dies after 25s
- Badly Injured: dies after 35s
- Injured: dies after 65s

Each of the soldiers spawns in a range of levels, so the amount of actual health is not consistent. So I am simply setting the starting health to the number of seconds needed (25/35/65%) and then reducing their health by 1% per second until they are dead. Simple and replicates WotLK Classic pretty much exactly. Without access to a real 3.3.5 server, I think this is as Blizzlike as I can make the dying time for each solider type.